dvc/command/add.py

```diff
@@ -9,7 +9,11 @@ class CmdAdd(CmdBase):
     def run(self):
         for target in self.args.targets:
```

Comment:
I have a controversial question - why don't we commit all the changed files by default?

Motivation - in most of the cases I'm sure the state is valid and I don't want to spend time on adding every single file to the param list. An alternative - additional option `--all-changed`.

Reply:
We do, you are just looking at CmdAdd implementation and not CmdCommit :)

Reply:
Good. Sorry, I have not noticed that.